src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/value_objects/ExportableEmails.java

```diff
@@ -0,0 +1,15 @@
+package de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+/**
+ * @author seiferla, Jonas-Karl
+ */
+public record ExportableEmails(
```

Comment:
```suggestion
public record ExportableEmail(
```

This is just one email, let's treat it as such :)